apps/storybook-react/stories/WalletHome.stories.tsx

```diff
@@ -7,6 +7,11 @@ import {
   BadgeCount,
   BadgeWrapper,
   BadgeNetwork,
+  Box,
+  BoxAlignItems,
+  BoxFlexDirection,
+  BoxFlexWrap,
+  BoxJustifyContent,
```

Comment:
Importing Box and Box types and replacing all `div`s with Box and Box props where applicable